particles/Tutorial/Kotlin/README.md

```diff
@@ -329,11 +357,20 @@ In essence, a store is where the data physically exists. The handle lets differe
 In the last tutorial, we set the Person handle inside of the particles. Person was also a singleton, it represented a single instance of the Person schema. In this tutorial, we will instead define a collection with multiple instances of Person inside the Arcs Manifest file. While a collection can be thought of as a list, there are no guarantees about the order of elements within the collection.
 Using some more sophisticated template interpolation, we can easily greet everyone in our collection. But that’s enough explanation, let’s get to some code!
 
-We begin with the Arcs manifest file which includes the collection of `PersonDetails`.
+We begin by adding an additional schema to the `PeopleSchemas.arcs` manifest. 
 ```
+//...
+
 schema PersonDetails
   name: Text
   age: Number
```

Comment:
I think this should include the entire PeopleSchemas.arcs file. Since it is a tutorial, I tend to err on the side of too much rather than too little.

Reply:
Cool. Good to make things consistent.